<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="7" passed="5" failed="2" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-04-30T23:10:15 SGT" name="All Test Suite" finished-at="2023-04-30T23:10:17 SGT" duration-ms="2338">
    <groups>
    </groups>
    <test started-at="2023-04-30T23:10:15 SGT" name="C:/Users/kalsh/IdeaProjects/APIAutomationFramework" finished-at="2023-04-30T23:10:17 SGT" duration-ms="2338">
      <class name="com.tmb.tests.GetRequests">
        <test-method signature="getEmployeeDetails()[pri:0, instance:com.tmb.tests.GetRequests@769f71a9]" started-at="2023-04-30T23:10:15 SGT" name="getEmployeeDetails" finished-at="2023-04-30T23:10:16 SGT" duration-ms="1304" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEmployeeDetails -->
        <test-method signature="getEmployee(java.lang.Integer,java.lang.String)[pri:0, instance:com.tmb.tests.GetRequests@769f71a9]" started-at="2023-04-30T23:10:16 SGT" name="getEmployee" data-provider="getData" finished-at="2023-04-30T23:10:16 SGT" duration-ms="376" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Sebastian]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEmployee -->
        <test-method signature="getEmployee(java.lang.Integer,java.lang.String)[pri:0, instance:com.tmb.tests.GetRequests@769f71a9]" started-at="2023-04-30T23:10:16 SGT" name="getEmployee" data-provider="getData" finished-at="2023-04-30T23:10:17 SGT" duration-ms="48" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Steve]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEmployee -->
        <test-method signature="getEmployee(java.lang.Integer,java.lang.String)[pri:0, instance:com.tmb.tests.GetRequests@769f71a9]" started-at="2023-04-30T23:10:17 SGT" name="getEmployee" data-provider="getData" finished-at="2023-04-30T23:10:17 SGT" duration-ms="39" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Ann]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEmployee -->
        <test-method signature="getEmployee(java.lang.Integer,java.lang.String)[pri:0, instance:com.tmb.tests.GetRequests@769f71a9]" started-at="2023-04-30T23:10:17 SGT" name="getEmployee" data-provider="getData" finished-at="2023-04-30T23:10:17 SGT" duration-ms="50" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[1232]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Shyam]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[[Name is ] 
expected: "Shyam"
 but was: null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: [Name is ] 
expected: "Shyam"
 but was: null
at com.tmb.tests.GetRequests.getEmployee(GetRequests.java:50)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getEmployee -->
      </class> <!-- com.tmb.tests.GetRequests -->
      <class name="com.tmb.tests.PostTests">
        <test-method signature="createEmployeeTest()[pri:0, instance:com.tmb.tests.PostTests@3c419631]" started-at="2023-04-30T23:10:17 SGT" name="createEmployeeTest" finished-at="2023-04-30T23:10:17 SGT" duration-ms="422" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[
expected: 201
 but was: 500]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: 
expected: 201
 but was: 500
at com.tmb.tests.PostTests.createEmployeeTest(PostTests.java:38)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:577)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createEmployeeTest -->
        <test-method signature="createEmployeeUsingExternalFile(java.lang.reflect.Method)[pri:0, instance:com.tmb.tests.PostTests@3c419631]" started-at="2023-04-30T23:10:17 SGT" name="createEmployeeUsingExternalFile" finished-at="2023-04-30T23:10:17 SGT" duration-ms="46" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.tmb.tests.PostTests.createEmployeeUsingExternalFile(java.lang.reflect.Method)]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createEmployeeUsingExternalFile -->
      </class> <!-- com.tmb.tests.PostTests -->
    </test> <!-- C:/Users/kalsh/IdeaProjects/APIAutomationFramework -->
  </suite> <!-- All Test Suite -->
</testng-results>
